core/objectUtils.js

```diff
@@ -31,6 +32,7 @@ export const isEqual = other => self => getUuid(other) === getUuid(self)
 // ====== READ
 export const getId = R.prop(keys.id)
 export const getUuid = R.propOr(null, keys.uuid)
+export const getOwnerName = R.propOr(null, keys.ownerName)
```

Comment:
I don't see getOwnerName as common prop. Add it only to SurveyInfo

Reply:
sure!